### ISSUE TITLE
preload the users assets in the AOI selector if the user connected its GEE account

### ISSUE THREAD

**Issue by 12rambau** [openforis/sepal#141]
When creating a recipe, instead of using a administrative border, the user can select an AOI with a EE table. If its GEE account is connected, would it be possible to use a Combobox instead of a textfield and preload the user Table assets ? 

We are doing using this behavior in the AssetField for modules: 
![Enregistrement de l’écran 2021-11-22 à 18 44 40](https://user-images.githubusercontent.com/12596392/142910079-879b4e4f-7470-4872-a40b-8ec2e1e0d2f1.gif)

**Comment by cdanielw** [comment]
Yes - improving the component for selecting assets is something we want to do. Searching through the user's assets and maybe even being the EE catalogue too.

**Comment by dfguerrerom** [comment]
this was already implemented
